src/libraries/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -748,6 +762,27 @@ public override int ReadByte()
             return bytesRead == 1 ? oneByte[0] : -1;
         }
 
+        public override unsafe int Read(Span<byte> buffer)
```

Comment:
Should the fallback paths in Read/WriteByte be updated to use the span-based APIs now that they're using this approach?